<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transformer</class>
 <widget class="QMainWindow" name="transformer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>transformer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="textInput"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="encryptButton">
         <property name="text">
          <string>Encrypt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decryptButton">
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Magic Word</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="magicWord"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textOutput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
